cfme/intelligence/reports/schedules.py

```diff
@@ -54,33 +55,45 @@ class SchedulesFormCommon(CloudIntelReportsView):
     name = TextInput(name="name")
     description = TextInput(name="description")
     active = Checkbox("enabled")
-    # Report Selection
-    filter1 = BootstrapSelectRetry("filter_typ")
-    filter2 = BootstrapSelectRetry("subfilter_typ")
-    filter3 = BootstrapSelectRetry("repfilter_typ")
-    # Timer
-    run = BootstrapSelect("timer_typ")
-    # Adding timer for hour, day, week, and zone because there is no single element
-    #  for the timer_interval.
-    timer_hour = BootstrapSelect("timer_hours")
-    timer_day = BootstrapSelect("timer_days")
-    timer_month = BootstrapSelect("timer_months")
-    timer_week = BootstrapSelect("timer_weeks")
-    time_zone = BootstrapSelect("time_zone")
-    starting_date = Calendar("miq_date_1")
-    hour = BootstrapSelect("start_hour")
-    minute = BootstrapSelect("start_min")
-    # Email
-    emails_send = Checkbox("send_email_cb")
-    from_email = TextInput(name="from")
-    emails = AlertEmail()
-    send_if_empty = Checkbox("send_if_empty")
-    send_txt = Checkbox("send_txt")
-    send_csv = Checkbox("send_csv")
-    send_pdf = Checkbox("send_pdf")
     # Buttons
     cancel_button = Button("Cancel")
 
+    @View.nested
+    class timer(View):                      # noqa
```

Comment:
There should only be two spaces before the `#` character for an in-line comment. These should not be so far to the right.